tools/ci_build/github/linux/ort_minimal/build_full_ort_and_create_ort_files.sh

```diff
@@ -23,7 +24,19 @@ python3 /onnxruntime_src/tools/ci_build/build.py \
     --build_wheel \
     --skip_tests \
     --enable_training_ops \
-    --enable_pybind --cmake_extra_defines PYTHON_INCLUDE_DIR=/opt/python/cp37-cp37m/include/python3.7m PYTHON_LIBRARY=/usr/lib64/librt.so
+    --enable_pybind --cmake_extra_defines PYTHON_INCLUDE_DIR=/opt/python/cp37-cp37m/include/python3.7m PYTHON_LIBRARY=/usr/lib64/librt.so \
+    --use_nnapi
+
+python3 /onnxruntime_src/tools/ci_build/build.py \
+    --build_dir /build --cmake_generator Ninja \
+    --config Debug \
+    --skip_submodule_sync \
+    --parallel \
+    --build_wheel \
+    --skip_tests \
+    --enable_training_ops \
+    --enable_pybind --cmake_extra_defines PYTHON_INCLUDE_DIR=/opt/python/cp37-cp37m/include/python3.7m PYTHON_LIBRARY=/usr/lib64/librt.so \
+    --use_coreml
```

Comment:
How about try to use the same build to have both `--use_nnapi` and `--use_coreml`